test/integration/parachain/staging/sequential/vaults.test.ts

```diff
@@ -114,123 +111,202 @@ describe("vaultsAPI", () => {
+                return;
+            }
+            
+            assert.isTrue(
+                premiumRedeemAmount.gte(issuableAmount),
+                `Amount available for premium redeem should be higher (${currencyTicker} vault)`
+            );
         }
-
-        const collateralCurrencyTyped = collateralCurrency as Currency<CollateralUnit>;
-
-        // The factor to adjust the exchange rate by. Calculated such that the resulting collateralization
-        // will be 90% of the premium redeem threshold. (e.g. 1.35 * 90% = 1.215)
-        const premiumRedeemThreshold = await interBtcAPI.vaults.getPremiumRedeemThreshold(collateralCurrency);
-        const modifyExchangeRateBy = premiumRedeemThreshold.mul(0.9).div(currentVaultCollateralization);
-
-        const initialExchangeRate = await interBtcAPI.oracle.getExchangeRate(collateralCurrencyTyped);
-        // crash the exchange rate so that the vault falls below the premium redeem threshold
-        const exchangeRateValue = initialExchangeRate.toBig().div(modifyExchangeRateBy);
-        const mockExchangeRate = new ExchangeRate<
-            Bitcoin,
-            BitcoinUnit,
-            typeof collateralCurrencyTyped,
-            typeof collateralCurrencyTyped.units
-        >(Bitcoin, collateralCurrencyTyped, exchangeRateValue);
-
-        // stub the oracle API to always return the new exchange rate
-        const stub = sinon.stub(interBtcAPI.oracle, "getExchangeRate")
-            .withArgs(sinon.match.any)
-            .returns(Promise.resolve(mockExchangeRate as any)); // "as any" to help eslint play nicely
-
-        const premiumRedeemVaults = await interBtcAPI.vaults.getPremiumRedeemVaults();
-
-        // Check that the stub has indeed been called at least once 
-        // If not, code has changed and our assumptions when mocking the oracle API are no longer valid
-        sinon.assert.called(stub);
-
-        // real assertions here
-        assert.equal(premiumRedeemVaults.size, 1);
-        assert.equal(
-            encodeVaultId(premiumRedeemVaults.keys().next().value),
-            encodeVaultId(vault_3_id),
-            "Premium redeem vault is not the expected one"
-        );
-
-        const premiumRedeemAmount = premiumRedeemVaults.values().next().value;
-        assert.isTrue(
-            premiumRedeemAmount.gte(issuableAmount),
-            "Amount available for premium redeem should be higher"
-        );
-    }).timeout(5 * 60000);
+    }).timeout(10 * 60000);
 
     it("should getLiquidationCollateralThreshold", async () => {
-        const threshold = await interBtcAPI.vaults.getLiquidationCollateralThreshold(collateralCurrency);
-        assert.equal(threshold.toString(), "1.1");
+        const expectedThresholdByTicker: Map<string, string> = new Map([
+            [Polkadot.ticker, "1.1"],
+            [Kusama.ticker, "1.1"],
+            [Kintsugi.ticker, "2"]
+        ]);
```

Comment:
Are these set during initialization? If not we should definitely review our use of magic numbers in tests so us changing the genesis on the parachain doesn't randomly break things here.

Reply:
No, those are magic numbers.
Happy to set during initialization - I couldn't find a way when working the tests. 

I think that piece of work would fit in well with the proposed changes by Dom here: https://github.com/interlay/interbtc-api/pull/415#discussion_r883290944